multicast/__init__.py

```
@@ -662,44 +662,44 @@
 
 try:
-	if """multicast.__main__""" in sys.modules:  # pragma: no cover
+	if "multicast.__main__" in sys.modules:  # pragma: no cover
 		global __main__  # skipcq: PYL-W0604
-		__main__ = sys.modules["""multicast.__main__"""]
+		__main__ = sys.modules["multicast.__main__"]
 except Exception:
 	import multicast.__main__ as __main__  # pylint: disable=cyclic-import - skipcq: PYL-R0401
 
-if __name__ in u'__main__':
+if __name__ in u"__main__":
```

Ruff (0.8.2)

Remove unicode literals from strings

Remove unicode prefix

(UP025)